tests/ui/specs/TimePicker/TimePicker-specs.js

```diff
@@ -0,0 +1,287 @@
+					timePicker.decrementer(timePicker.hour).click();
+					expect(timePicker.decrementer(timePicker.hour).isFocused()).to.be.true();
+					browser.pause(500);
+					const {hour: value} = extractValues(timePicker);
+					expect(value).to.equal(hour);
+				});
+			});
+		});
+
+		describe('disabled with \'defaultValue\'', function () {
+			const timePicker = Page.components.timePickerDisabledWithDefaultValue;
+			it('should not update \'defaultValue\' on click', function () {
+				const {hour, minute, meridiem} = extractValues(timePicker);
+
+				timePicker.decrementer(timePicker.minute).click();
+				expect(timePicker.decrementer(timePicker.minute).isFocused()).to.be.true();
+
+				timePicker.decrementer(timePicker.hour).click();
+				expect(timePicker.decrementer(timePicker.hour).isFocused()).to.be.true();
+
+				timePicker.decrementer(timePicker.meridiem).click();
+				expect(timePicker.decrementer(timePicker.meridiem).isFocused()).to.be.true();
+
+				browser.pause(500);
+				expect(hour).to.equal(12);
+				expect(minute).to.equal(0);
+				expect(meridiem).to.equal('AM');
+			});
+
+		});
+	});
+
+	describe('RTL locale', function () {
+		const timePicker = Page.components.timePickerDefault;
+
+		beforeEach(function () {
+			Page.open('?locale=ar-SA');
+		});
+
+		it('should focus middle picker (hour) when selected', function () {
+			browser.waitUntil(() => timePicker.decrementer(timePicker.hour).isFocused(), {timeout: 1500,  interval: 100});
+		});
+
+		it('should have minute-hour-meridiem order', function () {
+			browser.waitUntil(() => timePicker.decrementer(timePicker.hour).isFocused(), {timeout: 1500,  timeoutMsg: 'initial', interval: 100});
+			Page.spotlightRight();
+			browser.waitUntil(() => timePicker.decrementer(timePicker.minute).isFocused(), {timeout: 1500,  timeoutMsg: 'minute', interval: 100});
+			Page.spotlightLeft();
+			browser.waitUntil(() => timePicker.decrementer(timePicker.hour).isFocused(), {timeout: 1500,  timeoutMsg: 'hour', interval: 100});
+			Page.spotlightLeft();
+			browser.waitUntil(() => timePicker.decrementer(timePicker.meridiem).isFocused(), {timeout: 1500,  timeoutMsg: 'meridiem', interval: 100});
+		});
+	});
+
+	describe('24-hour locale', function () {
+		const timePicker = Page.components.timePickerWithDefaultValue;
+
+		beforeEach(function () {
+			Page.open('?locale=fr-FR');
+		});
+
+		it('should not have a meridiem picker', function () {
+			expect(timePicker.meridiem.isExisting(), 'meridiem exists').to.be.false();
+		});
+
+		it('should display hours in 24-hour format', function () {
+			expect(extractValues(timePicker).hour).to.equal(0); // midnight hour
+		});
+
+		it('should increment hours from 0 to 23', function () {
+			for (let i = 0; i < 24; i++) {
+				timePicker.incrementer(timePicker.hour).click();
+			}
+			const {hour} = extractValues(timePicker);
+			expect(hour).to.equal(23);
+		});
+
+		it('should decrement hours from 23 to 0', function () {
```

Comment:
I think these tests aren't meaningful here.
Let's add "increment hours from 23 to 0" and "decrement hours from 0 to 23" from PLAT-124934 with 'wrap' prop.

Reply:
Or we could just merge this and add additional ones from there.

Reply:
please merge this one and I'll update 124934 right after

Reply:
👍